CONTRIBUTING.md

```diff
@@ -40,7 +40,7 @@ Working on your first Pull Request? You can learn how from this *free* series [H
 | ------------------------------------------- | ------- |
 | [MongoDB Community Server](https://docs.mongodb.com/manual/administration/install-community/) | `~ ^3`  |
 | [MailHog](https://github.com/mailhog/MailHog) | `~ ^1` |
-| [Node.js](http://nodejs.org)                | `~ ^8`  |
+| [Node.js](http://nodejs.org)                | `~ ^8.9.3`  |
```

Comment:
I was told that some some changes were made that require a newer Node.js version than what we have in the requirements. Even v8.2.0 is too old. v8.9.3, the latest LTS, does work. That's why I changed it to that particular version.

Reply:
Ok, ignore the previous comment if that is the case. Thanks for letting me know.

Reply:
Users should always be using the latest, stable, or lts version of Node. v8.2.0 is not considered stable so its not supported; however, in order for us to not have to continuously update the documentation `^8` is an easy way to say `use Node.js version 8 (whatever the current stable version is).`

Reply:
@Ethan-Arrowood Node v8.x.x is LTS which what we use and support as per standard conventional norms. We do not support any other version. The notation ^8 itself means the latest compatible version installed on a system. Installers like nvm will default this to for example to `8.9.3` if no other compatible version is available, meaning if (`8.2.0` is installed, it will be used, by such installers).

We already mention a couple of lines below, that upgrading to latest releases is recommended.

Hence my argument for keeping this line as it is, originally.

That said, I do not want this discussion to be going in direction of bike-shedding which it seems, it has now begun to. 😅 !

So, lets just merge this. If there is an issue in dev on this particular version, we can always go back to my original proposal.

Reply:
In any case on production, we are always using the latest LTS.

Reply:
Thanks for the improvements @zk433 and for QA'ing @raisedadead!

@raisedadead I originally mentioned that this should be changed to Zuzana on Gitter. A couple of contributors were running into an issue where they would get an error because of an object spread operator in the seed scripts (added about two weeks ago). Most of these contributors were running Node.js v 8.2.0, and because that was new enough according to the docs, the source of the problem was difficult to find.

I do like @Ethan-Arrowood's suggestion of saying the latest LTS should be used, but this makes it a bit more difficult for contributors to check whether the version they're running should work or not. Stating a minimum version explicitly makes spotting issues caused by older versions of Node a lot easier.

Reply:
Sure thing. Lets keep whatever has been found to best working for everyone?